build/yamls/antrea-windows.yml

```diff
@@ -0,0 +1,197 @@
+apiVersion: v1
+data:
+  run-antrea-agent.ps1: |-
```

Comment:
Would you rename this one to Run-AntreaAgent.ps1?

Reply:
Sure, it makes sense. Done